plugins/configuration.rb

```diff
@@ -0,0 +1,92 @@
+module Jekyll
+  class ConfigurationBlock < Liquid::Block
+    TYPE_LINKS = {
+      'action'       => '/docs/automation/action/',
+      'device_class' => '/components/binary_sensor/',
```

Comment:
This is not unique to the binary sensor. Better would be to link to the component that the platform that is using device_class belongs to?

Reply:
I have added the ability to specify the component/platform in the form of `{% configuration binary_sensor.template %}` tag. If not specified, the component and platform will automatically be detected from `page['slug']`. This will allow us to use `%{component}` in the `TYPE_LINKS`.